packages/prisma-client/migrations-cli/README.md

```diff
@@ -0,0 +1,50 @@
+## CLI Reference
+
+### `--dev`
+
+This flag can be added to any command. It informs the CLI that it’s used in a development environment, which slightly changes its behaviour:
+
+- automatically loads variables from `.env` files,
+- uses a lockfile,
+- asks for confirmation before applying migrations,
+- allows resetting the database using the `reset` command.
```

Comment:
commented on this below, reset should have a different protection mechanism than implicit dev check

Reply:
Ok. Will resolve this comment as a duplicate